infrastructure/cdn-in-a-box/cache/Dockerfile

```diff
@@ -60,6 +60,7 @@ RUN dnf -y install epel-release && \
         libmaxminddb            \
         libtool-ltdl            \
         libunwind               \
+	libuuid			\
```

Comment:
please match indentation of the rest of the file. Currently, instead of real indentation this file uses spaces. If you'd prefer to instead convert this file to use real indentation, that's fine also.

Reply:
switched to spaces

wrt to the tests, we are seeing them internally but I believe they have not been enabled yet on open source, so this is in preparation of that. Also to provide future compatibility with ats 9.2 rpms

Reply:
"not ... enabled" meaning they aren't running in GHAs or haven't even been written yet?

Reply:
We're seeing failures with our internal build system with Rocky8, on the TO API Tests. We're not entirely sure why it's succeeding in Github, but failing on our internal build. 

The error is
```
infrastructure/cdn-in-a-box/docker-compose -p ATCN-toapi-7 -f docker-compose.traffic-ops-test.yml -f docker-compose.yml --no-ansi build --parallel --no-cache edge enroller dns db smtp trafficops integration

rpm -Uvh /trafficserver.rpm

 ---> Running in 4eaac11a7b28
error: Failed dependencies:
	xz is needed by trafficserver-13492:9.1.2-22.b965ea10a.el8.x86_64
```

It looks like the `rpm` call doesn't install dependencies. I'm not sure how it's succeeding in Github here.

Every PR run I look at is using a cached RPM. Is it possible a clean build in Github that didn't use the cache would also fail?

We're still investigating.

Reply:
> It looks like the rpm call doesn't install dependencies. I'm not sure how it's succeeding in Github here.

It uses rpm to install the trafficserver RPM in order to assert that all of the dependencies are installed in a previous docker layer. This is preferable for users of CDN in a Box in order to reduce the build time when the trafficserver RPM changes, as
* `rpm` runs much faster than `yum`
* the same dependencies don't need to be repeatedly reinstalled.

> Every PR run I look at is using a cached RPM. Is it possible a clean build in Github that didn't use the cache would also fail?

While it is true that the ATS RPM is usually cached, the cache key includes the latest release tag and latest commit on the release branch used (currently `8.1.x`), so it does update whenever the `8.1.x` release branch does. For example, from GHA run [7676647738](https://github.com/apache/trafficcontrol/runs/7676647738) of the *trafficserver* job of the * TC Health Client Integration Tests* GHA workflow:

```
 Cache restored from key: trafficserver-8.1.4-3.61478f21e.el8.x86_64.rpm-13333c2156bae36275075764d90aabd12b94c622db675d823b68cd3ca382cfb9
```

from
https://github.com/apache/trafficcontrol/blob/f08435a625fd007198b9e993f47d4cdb9c9821d5/.github/workflows/cache-config-tests.yml#L122 https://github.com/apache/trafficcontrol/blob/f08435a625fd007198b9e993f47d4cdb9c9821d5/.github/workflows/cache-config-tests.yml#L122 and https://github.com/apache/trafficcontrol/blob/f08435a625fd007198b9e993f47d4cdb9c9821d5/.github/workflows/health-client-tests.yml#L123

> ```
> 	xz is needed by trafficserver-13492:9.1.2-22.b965ea10a.el8.x86_64
> ```

That would be nice if the GHAs used an ATS 9 RPM instead. Since #7003 moves us in that direction, I'm all for merging it, but [the existing ATS RPM SPEC file](https://github.com/apache/trafficcontrol/blob/f08435a625/cache-config/testing/docker/trafficserver/trafficserver.spec) fails to build for ATS 9 (IIRC it said something.yml did not exist), so that will need to be updated, too. I'll open a proper issue for that.

Reply:
I made a test PR to not use the cache, and it succeeded:
https://github.com/apache/trafficcontrol/runs/7678948197?check_suite_focus=true
```
 Wrote: /root/rpmbuild/RPMS/x86_64/trafficserver-8.1.4-3.61478f21e.el8.x86_64.rpm
 ```

So, not sure what the difference is with our internal build. Will continue to investigate

Reply:
Opened #7006 for ATS 9.1.x failing to build